app/models/email_audit.rb

```diff
@@ -3,7 +3,8 @@ class EmailAudit < ApplicationRecord
   belongs_to :school, optional: true
 
   validates :message_type, presence: true
-  validates :template, presence: true
 
   scope :sent_to, ->(email_address) { where(email_address: email_address) }
+
+  scope :problematic, -> { where(govuk_notify_status: %w[permanent-failure temporary-failure technical-failure]) }
```

Comment:
not used at the moment but i envisage the emails audits web interface to show problematic emails only in the near future so they can be triaged and actioned